app/javascript/components/time.jsx

```diff
@@ -32,7 +32,7 @@ const TimeValue = styled(Row)`
 
 export class Time extends React.Component {
   static propTypes = {
-    timezone: PropTypes.string.isRequired,
+    timezone: PropTypes.string,
```

Comment:
why is this component changing?

Reply:
I believe `.isRequired` can remain on the `timezone` property because the TimeHero component will return the LoadingMessage or ErrorMessage components if the location property doesn't contain data (so the Time component will never be rendered if location doesn't have the required data).

Reply:
I removed `.isRequired` to avoid the error message: "The prop `timezone` is marked as required in `Time`, but its value is `undefined`" for all locations. Am I not declaring `timezone` properly?

Reply:
Is it possible that `loading` and `error` are both false at some point?